subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/InstantExecutionException.kt

```diff
@@ -69,8 +71,8 @@ open class InstantExecutionProblemsException : InstantExecutionException {
         problems: List<PropertyProblem>,
         htmlReportFile: File
     ) : this(
-        "Instant execution problems found in this build.\n" +
-            "Failing because -D${SystemProperties.failOnProblems} is 'true'.",
+        "Configuration cache problems found in this build.\n" +
+            "Gradle can be made to ignore these problems via --${ConfigurationCacheOption.LONG_OPTION}=warn.",
```

Comment:
I'm not sure I'd hard code only one of several ways for ignoring the problems in the console output. It might be better to link to the appropriate section in the user manual (in a follow up PR).